packages/angular_devkit/schematics/src/tree/scoped.ts

```diff
@@ -125,15 +135,61 @@ export class ScopedTree implements Tree {
   }
 
   apply(action: Action, strategy?: MergeStrategy): void {
-    return this._base.apply(action, strategy);
+    return this._base.apply(this._fullPathAction(action), strategy);
+  }
+
+  get actions(): Action[] {
+    const scopedActions = [];
+
+    for (const action of this._base.actions) {
+      if (!action.path.startsWith(this._root.scope + '/')) {
+        continue;
+      }
+
+      if (action.kind === 'r') {
+        if (action.to.startsWith(this._root.scope + '/')) {
+          scopedActions.push({
+            ...action,
+            path: join(NormalizedRoot, relative(this._root.scope, action.path)),
+            to: join(NormalizedRoot, relative(this._root.scope, action.to)),
+          });
+        }
+
+        continue;
```

Comment:
It is. We need to handle the case when the action is not rename.  Rename is just special since it has two paths that need to be checked and potentially modified.

Reply:
Ah yes, you're right. Please ignore the latter remark then.
If we remove `continue` here and combine the first two if conditions the code will be less confusing.

Reply:
just pushed.  should be better now :)

Reply:
LGTM, thank you! :)